src/main/resources/schema/user.graphqls

```diff
@@ -1,7 +1,8 @@
 type User {
     id: ID!
     username: String!
-    email: String!
+    emailSearch: String!
+    emailDisplay: String!
```

Comment:
If we don't need to expose the emailSearch value then we can keep this named "email".